mssql-config.yml.sample

```
@@ -35,6 +35,8 @@ integrations:
   labels:
     env: production
     role: mssql
+    # the label below is so there can be a common attribute to filter on within the dashboard. Most useful for multi-server environments
+    db_hostname: Enter-Custom-Hostname-Here    
```

Hi @lchapman4,

Since it is a configuration example useful for a particular use case(multi-server environments), would it make sense to have it commented by default?

On the other hand, considering what @paologallinaharbur [said](https://github.com/newrelic/nri-mssql/pull/129/files#r1278968243) if its usage is really common, it could make sense to create a Feature Request to include it as an argument.

Let me know what you think about the suggestion.

```suggestion
    # db_hostname: my-custom-hostname # useful to filter in dashboards, especially in multi-server environments.   
```

---

I like it. The comments help explain what we're trying to do. Thanks @sigilioso! 

It would be great if we could incorporate something like this as an option natively in the product. More and more customers typically use the database OHIs to monitor multiple instances because they either can't install the OHI locally (cloud-managed instance, ie RDS) or they don't have permissions due to local security policies. This is in addition to enterprises running the OHI in a hub-and-spoke model where one dedicated OHI host will monitor several database instances to reduce the overhead on the database hosts. Something similar to our [display_name setting in the infra agent](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#display-name) but for remote instances would be great.

---

As it is that common, maybe it makes sense to open a Feature Request so PMS can keep track of it and potentially prioritize its addition.
Thanks again for improving the example! :)